deploy/apko.yaml.tmpl

```diff
@@ -0,0 +1,58 @@
+contents:
+  repositories:
+    - https://packages.wolfi.dev/os
+    - ./packages/
+  keyring:
+    - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
```

Comment:
```suggestion
    - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
    - ./packages/chainguard-enterprise.rsa.pub
```

Reply:
same error when adding to the apko keyring. i tried adding it to the melange keyring and got this error
```
Error: failed to build package: unable to build guest: unable to create build context: initializing apk: failed to initialize apk keyring: failed to read apk key: open ./packages/chainguard-enterprise.rsa.pub: no such file or directory
2023/11/29 15:41:51 error during command execution: failed to build package: unable to build guest: unable to create build context: initializing apk: failed to initialize apk keyring: failed to read apk key: open ./packages/chainguard-enterprise.rsa.pub: no such file or directory
```